src/util/sys/mod.rs

```diff
@@ -0,0 +1,32 @@
+#[cfg(target_os = "linux")]
+use libc;
+#[cfg(target_os = "linux")]
+use std::io::Error;
+
+pub enum AdjustPriType {
```

Comment:
```
pub enum PriorityType {
    High,
    Normal,
    Low
}

pub fn set_priority()
```
IMO, adjust with Incr and Decr confuses me, calling `adjust` witch Incr doesn't mean increasing priority each time, so using `set` with High or Low here is better here.